packages/app/components/checkout/checkout-claim-form.tsx

```diff
@@ -117,17 +117,17 @@ const CheckoutFormLayout = ({
   });
   const router = useRouter();
   const paymentMethods = usePaymentsManage();
-  const defaultPaymentMethod = useMemo(
-    () => paymentMethods.data?.find((method) => method.is_default),
-    [paymentMethods.data]
-  );
   const paymentMethodsList = useMemo(
     () => uniq(paymentMethods.data?.filter((f) => f.type == "card")),
     [paymentMethods.data]
   );
-  const [savedPaymentMethodId, setSavedPaymentMethodId] = useState(
-    defaultPaymentMethod?.id
+  const defaultPaymentMethod = useMemo(
+    () => paymentMethods.data?.find((method) => method.is_default),
+    [paymentMethods.data]
   );
+  const [savedPaymentMethodId, setSavedPaymentMethodId] = useState<
+    undefined | string
+  >(defaultPaymentMethod?.id);
```

Comment:
Do we need to call `setSavedPaymentMethodId` when the `paymentMethods` API call succeeds to set the initial state?